lib/game.ex

```diff
@@ -76,10 +76,10 @@ defmodule Game do
 
   def status(%Game{players: players, turns: turns}) do
     cond do
-      player_won?(turns[players.p1.token]) ->
-        {:ended, {:winner, players.p1.name}}
-      player_won?(turns[players.p2.token]) ->
-        {:ended, {:winner, players.p2.name}}
+      player_won?(turns[players.player_one.token]) ->
```

Comment:
What do lines 79 and 81 do? I can't really grok these lines...

Reply:
turns[players.player_one.token] returns a maps with all the moves made by player_one. player_won? will check the moves to figure out if player_one has a winning pattern. if it does have a winning pattern, it'll return the status of player_one won the game. Likewise for player_two